### PR TITLE
SeleniumConf Chicago 2023 Update Blog Post

### PR DESCRIPTION
**Pull request by testbot206**
Post announcing confirmed speakers and keynotes, plus teasing Fix a Bug, Become a Committer invite-only workshop and scholarship program. This needs to be reviewed and vetted by the organizing committee at large.

**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Types of changes

- [x] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Comment by testbot206** [comment]
Sorry, closed by mistake. I am a stranger in a strange land...

File: website_and_docs/content/blog/2022/seleniumconf-chicago-2023-update [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+### Announcing the SeleniumConf Chicago 2023 Scholarship Program
+
+The mission of the Selenium project is to educate the QA and developer commmunities about Selenium, the importance of quality assurance, and automated testing in general. Both the Selenium project and the SeleniumConf organizers very much want everyone to be able to attend the conference, but we know that not everyone’s personal or financial circumstances make this possible - so, we're setting aside a number of free tickets, and covering travel expenses, as part of our SeleniumConf Chicago 2023 scholarship program.
+
+The program aims to support individuals from underrepresented groups in tech and those facing economic or social hardship so they have access to SeConf content and networking opportunities. If you’re an aspiring software professional looking to use Selenium, or a current one without access to the funds to buy your ticket, you should apply. We will award tickets to people who can show that the knowledge gained will be useful for their ongoing career. The application form will be posted in January, 2023.
+
+### Ready to Go, But Your Manager Needs MOre Convincing?
```
Comment (harsha509) [inline website_and_docs/content/blog/2022/seleniumconf-chicago-2023-update]: Minor typo `More`

Comment (testbot206) [reply website_and_docs/content/blog/2022/seleniumconf-chicago-2023-update]: Thanks, fixed plus a few more!

**Review by maaretp — APPROVED** [review]
This looks good to me as text. I was wondering if we wanted to mention the committers coming together at this event aspect, and add energy around the social gathering of the community and welcoming nature of new people to the community?

**Pull request merged**